prowler/lib/check/custom_checks_metadata.py

```diff
@@ -18,9 +18,63 @@
                         "Severity": {
                             "type": "string",
                             "enum": valid_severities,
-                        }
+                        },
+                        "CheckTitle": {
+                            "type": "string",
+                        },
+                        "Description": {
+                            "type": "string",
+                        },
+                        "Risk": {
+                            "type": "string",
+                        },
+                        "RelatedUrl": {
+                            "type": "string",
+                        },
+                        "Remediation": {
+                            "type": "object",
+                            "properties": {
+                                "Code": {
+                                    "type": "object",
+                                    "properties": {
+                                        "CLI": {
```

Comment:
Can you add some tests to modify for example the `Remediation.Code.CLI` and check that the rest of the original check's metadata is kept.

Reply:
Done, also I've modified the `update_check_metadata` method since It wasn't working fine for sub-fields.